Program.cs

```diff
@@ -1,2 +1,58 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using System;
+
+namespace CMP1124M_Assignment
+{
+    public class Program
+    {
+        private const int NumberOfShares = 3;
+        public static void Main(string[] args)
+        {
+            int[][] shareValues256 = new int[NumberOfShares][];
```

Comment:
Should move certain 256 vs 2048 specific code to separate methods and have other repeated code in other methods :)